### ISSUE TITLE
negative numbers?

### ISSUE THREAD

**Issue by ak-hash** [HelenFoster/AnkiAddons#12]
Hi there I'm not sure if this is a glitch or what .. but recently I've been seeing -ve numbers on the stats. what does that mean? 
<img width="1280" alt="Screen Shot 2021-09-21 at 5 27 49 PM" src="https://user-images.githubusercontent.com/56414365/134265025-e4ae596c-7c8b-4be2-b7e0-a0076b9a7734.png">

**Comment by HelenFoster** [comment]
Hmm, yes, I see it.

**Comment by DataCollectionPoint528** [comment]
The issue is the left field in Card has been changed in a recent update to simply record the number of learning steps left until graduation for a card rather than recording both the number of steps left today and overall number for graduation - no idea why they decided to do this.

**Comment by HelenFoster** [comment]
Yes, you're right. There now seem to be 3 version of the scheduler. I wonder if anyone is still using the oldest one.
